jsx/Panel.js

```diff
@@ -46,21 +46,23 @@ class Panel extends Component {
     // Add panel header, if title is set
     const panelHeading = this.props.title ? (
       <div
-        className="panel-heading"
+        className='panel-heading'
         onClick={this.toggleCollapsed}
-        data-toggle="collapse"
+        data-toggle={this.props.collapsing ? 'collapse' : ''}
         data-target={'#' + this.props.id}
-        style={{cursor: 'pointer'}}
+        style={this.props.collapsing ?
+          {cursor: 'pointer'} : {cursor: 'default'}
+        }
       >
-        {this.props.title}
-        <span className={glyphClass}></span>
+        <h3 className={'panel-title'}>{this.props.title}</h3>
```

Comment:
this added `<h3>` might be why the glyphicon gets bumped down? not too sure but regardless, this style change looks ugly on the other modules. could we keep it to just `{this.props.title}`?

Reply:
i do see why you need this added header tag for the login module though. could we pass another prop called "bold" or something?

Reply:
@zaliqarosli awesome fixed.